iridauploader/model/sequencing_run.py

```diff
@@ -34,20 +34,15 @@ class SequencingRun:
                 }
             }
         },
-        '_assemblies': {
-            'type': 'boolean',
-            'required': True
-        },
```

Comment:
Was this just an unused property on the sequencing run?  No need for adding the upload mode here?

Reply:
Previously when I implemented assemblies, My thought process was that if the files on a `SequencingRun` were assemblies, we should denote that on the object, s.t. when it uploads, the `api_handler` queries the `SequencingRun` for file type, and chooses the appropriate api call to make.

Now that we have `default` (illumina), `assemblies`, and `fast5` files, I rethought how it should be handled, and decided the code flow would be cleaner/more logical to remove the additional steps of appending a file type to the `SequencingRun` based off of user input, and immediately fetching it again in the handler, and instead have the user option go directly to the handler. 

The reason this is cleaner is because it removes Model dependent logic from the `api` layer, and leaves the api to just do api things. Additionally it simplifies the handler logic, so it doesn't need to append/fetch variables to/from the model when it already has the data on the same layer.

In short: Made the simple solution with the assemblies branch because it was a single case. Now that it's more complex, refactor to make it generic.

Reply:
Sounds good, thanks for the full description.